### ISSUE TITLE
descs: reflect relative/absolute property of control

### ISSUE THREAD

**Issue by sensestage** [ModalityTeam/Modality-toolkit#144]
e.g. for mouse wheels, encoder knobs

**Comment by adcxyz** [comment]
how about adding this in descriptions: 
(
elementType: \encoder, 
valueSetType: \relative, // and default is \absolute
...
)

**Comment by LFSaw** [comment]
for now, I thought there is

```
\encoder -> relative rotary
\knob -> absolute rotary
```

which is not really a good idea... 

maybe 

```
elementType: \rotary,
valueType: \relative,  // and default is \absolute
```

?

**Comment by adcxyz** [comment]
added valueType: \relative to all mouse descs, so the info is present.
I am not convinced that accumulating values in derived elements is useful; 
it is more flexible to do that in the destination controller, which has its own state 
already, and set its value only with RelSet.
closed with 5e02afe.
